assets/js/components/Collection/ListRow.jsx

```diff
@@ -48,7 +48,7 @@ const CollectionListRow = ({ collection }) => {
               </Link>
             </p>
             <p className="control">
-              <button className="button" onClick={e => onOpenModal(e, project)}>
+              <button className="button" onClick={() => onOpenModal(id, name)}>
```

Comment:
So, if i did onClick={openModal({id, name})} -- it goes into infinite loop and if I do onClick="openModal({id, name})" this obviously fails coz its getting a string, is there some other way to do this? Like use `` instead of "" or {}?